lib/screens/terms_and_conditions/screen.dart

```diff
@@ -89,24 +76,24 @@ class _TermsAndConditionsScreenState extends State<TermsAndConditionsScreen> {
                 Flexible(
                   child: GestureDetector(
                     onTap: () {
-                      _launchUrl(widget.viewModel.currentTac.url);
+                      _launchUrl(widget.validTermsAndConditions.url);
```

Comment:
A suggestion would be to use `Consumer` and then inject state. Example could be
```
        Consumer<TermsAndConditionAcceptance>(
            builder: (context, tacState, child) {
          return Column(
            children: [
              Row(
                children: [
                  Flexible(
                    child: GestureDetector(
                      onTap: () {
                        _launchUrl(tacState.valid!.termsAndConditions.url);
...
```

Then we can avoid given variables in constructors.